### PR TITLE
Changed variable names to make code more readable

### PR DESCRIPTION
**Pull request by joshniemela**
Changed various variable names so that they are more intuitive and so that the code is more readable overall. This PR only changes variable names internally and doesn't contain any breaking changes.

### CODE REVIEW

File: canvas.fs [hunk 1]
```diff
@@ -131,35 +158,39 @@ let getKey (k:key) : ImgUtilKey =
...
+let runApp (title:string) (width:int) (height:int)
            (draw: int -> int -> 's -> canvas)
            (onKeyDown: 's -> key -> 's option) (s:'s) : unit =
 
     let state = ref s
 
     SDL.SDL_Init(SDL.SDL_INIT_VIDEO) |> ignore
 
-    let viewWidth, viewHeight = w, h
+    let view, height = width, height
```
Comment (madsobitsoe) [inline canvas.fs:169]: "view" and "width" do not seem to be the same thing. 
"viewWidth" refers to the width of the window we want SDL to create, viewHeight likewise the height. 

I would recommend changing this line to `let viewWidth, viewHeight = width, height`

Comment (joshniemela) [reply canvas.fs]: Not entirely sure how that happened but it should be fixed now

**Comment by joshniemela** [comment]
Should I add an additional commit containing the dotnet config with Fantomas-fmt so that everyone has the same formatting tool?

**Comment by kfl** [comment]
Thank for the contribution.

This PR is no longer relevant for Canvas 2.0 because the internals have been completely reworked.

**Pull request closed without merging**